metadata/index.html

```diff
@@ -318,37 +318,41 @@
             </p>
             <dl>
               <dt><code>_row</code></dt>
-              <dd><code>_row</code> is set to the row number of the row that is   currently being processed
+              <dd><code>_row</code> is set to the row number of the row that is currently being processed
                 <div class="issue" data-number="32">
                   Where does row numbering begin?
                 </div>
               </dd>
+              <dt><code>_name</code></dt>
+              <dd><code>_name</code> is set to the <a>property value</a> of the <code>name</code> property on the cell column that is currently being processed.
+              </dd>
               <dt>column names</dt>
               <dd>a variable is set for each column within the schema; the name of the variable is the percent-encoded name of the column and the value is the canonical representation of the value of the cell in that column in the row that is currently being processed</dd>
             </dl>
+            <p>The <a>property value</a> of a <a>URI template property</a> is only valid while converting tables as defined in <a href="#converting-tables" class="sectionRef"></a>. The value is the result of applying the template for against the current row and resolving against the <a>base URL</a> as a <a>link property</a> if the metadata value is defined, or the default value if defined; otherwise the value is <code>null</code>.</p>
```

Comment:
I'm not sure what "...is only valid..." means in this sentence. Do you mean "is only relevant"?

I don't understand what "if the metadata value is defined, or the default value if defined" is referring to? And under what circumstances can the value end up being `null`?

Perhaps add another example (or adapt an existing one) showing resolution against the base URL?

Also s/template for against/template against/